api/agent/pure_runner.go

```diff
@@ -668,6 +784,16 @@ func PureRunnerWithAgent(a Agent) PureRunnerOption {
 	}
 }
 
+func PureRunnerWithStatusImage(imgName string) PureRunnerOption {
```

Comment:
Thanks for adding the note to options.md.  That does point out where the documentation for save and load reside, but it doesn't make it clear that the correct operation of the status image is dependent upon loading an image with tags correctly embedded.  Would you please add a comment here, or somewhere else if this isn't the appropriate place, that makes this clear.  I'm trying to avoid having somebody spend a lot of time debugging a runner that won't load a status image, simply because the image loaded isn't bearing any tags.  Something like the following would suffice, assuming the description is factually accurate:
```
// PureRunnerWithStatusImage returns a PureRunnerOption that annotates a PureRunner with a
// statusImageName attribute.  This attribute names a file that is loaded by docker when a status check
// gRPC message is received by the runner.  Correct operation of the status image requires that an image be
// loaded by the DOCKER_LOAD_FILE environment variable.  In addition, this docker image must have been
// created with the tag name that is passed as the argument in imgName.  See docs/operating/options.md for
// additional details.
```

Reply:
Added comments to clarify the PureRunnerWithStatusImage() usage.